src/test/resources/org/prebid/server/it/floors/provided-floors.json

```diff
@@ -0,0 +1,15 @@
+{
+  "floorMin": 1000,
+  "floorProvider": "",
+  "skipRate": 0,
+  "data": {
+    "skipRate": 0,
+    "modelGroups": [
+      {
+        "values": {
+          "banner": 10
```

Comment:
Would be better to have more rules, not for only one but for multiple fields, to check priority and different field types resolving